test/auro-input.test.js

```diff
@@ -212,6 +212,198 @@ describe('auro-input', () => {
 
     await expect(el).to.be.true;
   });
+
+  it('handles credit card formatting - starts with "4" is Visa', async () => {
+    const el = await fixture(html`
+      <auro-input id="format-ccWithIcon" format='{"creditCard": true, "icon": true}' icon="true" label="Credit Card Number with Icon" required></auro-input>
+    `);
+
+    await expect(el).to.be.accessible();
+
+    const input = el.shadowRoot.querySelector('input');
```

Comment:
Can we move this query inside the setInputValue helper? That way we don't have to do it in every test.

Reply:
This is good thinking. Also going to simplify the code for cc card icon validation into a function rather than repeat it in each test.